tools/topology/topology2/platform/intel/sdw-jack-generic.conf

```diff
@@ -0,0 +1,156 @@
+#
+# List of all DAIs
+#
+#ALH Index: 0, Direction: duplex
+Object.Dai.ALH [
+	{
+		dai_index 0
+		id 		$SDW_JACK_OUT_BE_ID
+		direction	"playback"
+		name		$SDW_JACK_OUT_STREAM
+		default_hw_conf_id	0
+		rate			48000
+		channels		2
+
+		Object.Base.hw_config.1 {
+			id	0
+			name	"ALH2"
+		}
+	}
+	{
+		dai_index 10
+		id 		$SDW_JACK_IN_BE_ID
+		direction	"capture"
+		name		$SDW_JACK_IN_STREAM
+		default_hw_conf_id	1
```

Comment:
there's only 1 hw_config. Why should the default_hw_config_id be 1 here?

Reply:
> there's only 1 hw_config. Why should the default_hw_config_id be 1 here?

Changed it to 0 now.